test/test_video.py

```diff
@@ -5,12 +5,14 @@
 import unittest
 import random
 
+import itertools
+
 
 import numpy as np
 
 import torch
 import torchvision
-from torchvision.io import _HAS_VIDEO_OPT, Video
+from torchvision.io import _HAS_VIDEO_OPT, VideoReader
```

Comment:
Would it make sense to add this as a conditional import (i.e. under `if _HAS_VIDEO_OPT`) to avoid potential build issues?

Reply:
The test is guarded by a
```python
@unittest.skipIf(_HAS_VIDEO_OPT is False, "Didn't compile with ffmpeg")
```
so I think this should be safe.